tasks/tasks/mixmasta_processors.py

```diff
@@ -319,7 +333,44 @@ def run_model_mixmasta(context, *args, **kwargs):
     # Final cleanup of temp directory
     shutil.rmtree(localpath)
 
-    response = {
-        "mixmaster_annotations": mm_ready_annotations
-    }
+    response = {"mixmaster_annotations": mm_ready_annotations}
     return response
+
+
+def scale_features(context, filename=None):
+    # 0 to 1 scaled dataframe
+    uuid = context["uuid"]
+    data_paths = context["dataset"]["data_paths"]
+    paths_to_process = [
+        path for path in data_paths if "_str" not in path and "_normed" not in path
+    ]
+    data_paths = [path for path in data_paths if "_normed" not in path]
+
+    data_paths_normalized = []
+    for path in paths_to_process:
+        file_name = path.split("/")[-1].split(".parquet")[0]
+        file_out_name = f"{file_name}_normalized.parquet.gzip"
+        dataframe = pd.read_parquet(path)
+        dataframe_scaled = mix.scale_features(dataframe)
+
+        s3_filepath = os.path.join(
+            settings.DATASET_STORAGE_BASE_URL, "normalized", uuid, file_out_name
+        )
+
+        file_buffer = io.BytesIO()
+
+        dataframe_scaled.to_parquet(file_buffer)
+        file_buffer.seek(0)
+
+        put_rawfile(path=s3_filepath, fileobj=file_buffer)
+        data_paths_normalized.append(str(s3_filepath))
+    update_indicator(uuid=uuid, data_paths_normalized=data_paths_normalized)
+
+    return
+
+
+def update_indicator(uuid, data_paths_normalized):
```

Comment:
This could be the only real change on this file. I think I was trying out a plugin and the quotes were all replaced to double quotes. I'll revert that the main change to be obvious here (PR).

Reply:
Reverted PR that introduced unrelated changes.